CHANGELOG.md

```diff
@@ -9,15 +9,21 @@
 - Add functionality to transform news into thread
 - The cache of the browser used now saves the states of the navigation sidebar. Hidden sub-spaces will stay hidden.
 - Spaces in sidebar are now sorted without considering emoji
+- Kanban card color is now the background of the card instead of the border and default text color adapt to the background color.
```

Comment:
```suggestion
- Kanban card color is now the background of the card instead of the border. The text color will adapt to the background color in order to stay legible.
```